src/main/java/Task.java

```diff
@@ -0,0 +1,40 @@
+public class Task {
+
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public Task(String description, boolean isDone) {
+
+        this.description = description;
+        this.isDone = isDone;
+    }
```

Comment:
Is this Constructor overload really necessary? From my understanding, all Tasks created should, by default, not be completed yet.

Reply:
Initially all tasks are created by default has not having been completed, but when you pass a task to mark() method then we also need to provide the boolean of the task has been done.

Reply:
I couldn't find any `mark()` methods, but if you are referring to
```java
public void done(boolean done)
``` 
on line 21 of your `Task` class, there isn't any `Task` constructors being passed into the method. You are simply modifying the `isDone` boolean class field to become true. I doubt there is an instance where you have to create a new `Task` object just to change the class field to be true.